app/models/user.rb

```diff
@@ -1,2 +1,6 @@
 class User < ActiveRecord::Base
+  include Authem::User
+
+  validates_presence_of :email, :password
+  validates_uniqueness_of :email
```

Comment:
Anthem comes with model validation so I removed it. I was outputting duplicate errors in the views.